### PR TITLE
Adds Windows Firewall Rules

### PR DESCRIPTION
**Pull request by steviecoaster**
Returns firewall data to PDQ Inventory. I don't remember if this is already surfaced or not. If it is, just close this :)

### CODE REVIEW

**Review by bryan-pdq — DISMISSED** [review]

File: PowerShell Scanners/Windows Firewall Rules/Scan Profile.xml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    <Collections type="list" />
+    <Scanners type="list">
+      <Scanner>
+        <ModifiedDate>2020-04-02T12:12:46.0000000-06:00</ModifiedDate>
+        <Name>Windows Firewall Rules</Name>
+        <Script></Script>
+        <FileName>C:\PowerShell-Scanners\PowerShell Scanners\Windows Firewall Rules\Windows Firewall Rules.ps1</FileName>
+        <Parameters></Parameters>
+        <AdditionalFiles></AdditionalFiles>
+        <RowLimit value="100" />
```
Comment (Colby-PDQ) [inline PowerShell Scanners/Windows Firewall Rules/Scan Profile.xml]: This needs to be higher. Even when I limit it to just enabled rules I have over 200.

File: PowerShell Scanners/Windows Firewall Rules/Windows Firewall Rules.ps1 [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+)
+
+process{
+    
+    if([version](Get-WmiObject win32_operatingsystem).Version -lt 6.3.0){
+        throw "This scanner only available on Server 2012/Windows8 or higher"
+    }
+    
+    switch($Enabled){
+        $true { Get-NetFirewallRule -Enabled True| Select-Object Name,DisplayName,Description,Profile,Direction,Action }
```
Comment (Colby-PDQ) [inline PowerShell Scanners/Windows Firewall Rules/Windows Firewall Rules.ps1:15]: I'd like to see a space after `True` to match the formatting of the line below.

File: PowerShell Scanners/Windows Firewall Rules/Windows Firewall Rules.ps1 [hunk 3]
```diff
@@ -0,0 +1,19 @@
+[cmdletBinding()]
+param(
+    [parameter()]
+    [switch]
+    $Enabled
+)
+
+process{
+    
+    if([version](Get-WmiObject win32_operatingsystem).Version -lt 6.3.0){
```
Comment (Colby-PDQ) [inline PowerShell Scanners/Windows Firewall Rules/Windows Firewall Rules.ps1]: The PowerShell Scanner is only compatible with PowerShell 3+, so you can use `Get-CimInstance` if you want.

File: PowerShell Scanners/Windows Firewall Rules/Windows Firewall Rules.ps1 [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+        throw "This scanner only available on Server 2012/Windows8 or higher"
```
Comment (Colby-PDQ) [inline PowerShell Scanners/Windows Firewall Rules/Windows Firewall Rules.ps1]: Please add "is" after "scanner", and a space between "Windows" and "8".

**Review by Colby-PDQ — CHANGES_REQUESTED** [review]

File: PowerShell Scanners/Windows Firewall Rules/Windows Firewall Rules.ps1 (hunk 2 above)
Comment (steviecoaster) [reply PowerShell Scanners/Windows Firewall Rules/Windows Firewall Rules.ps1:15]: Fixed!

File: PowerShell Scanners/Windows Firewall Rules/Windows Firewall Rules.ps1 (hunk 3 above)
Comment (steviecoaster) [reply PowerShell Scanners/Windows Firewall Rules/Windows Firewall Rules.ps1]: Fixed!

**Comment by steviecoaster** [comment]
There we go. Latest commit should address all requested changes.

**Review by Colby-PDQ — APPROVED** [review]
Looks good!

**Pull request merged**